### ISSUE TITLE
remove phenotypes "mononucleate vegetative cell with mislocalized nucleus"

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3252]
<img width="277" alt="remove phenotypes" src="https://user-images.githubusercontent.com/7359272/168294372-da61c124-e2a4-489b-a8eb-e1a7cf5e2a22.png">

@manulera  pointed out these are misleading.

**Comment by manulera** [comment]
This refered to this experiment where they show that in a background where mitosis fails (different reasons tested), a diploid cell which inherits the undivided nucleus is formed, because actin displaces the nucleus. Some actin mutants may prevent this displacement of the nucleus from happening and the chromosomes are cut:

![](https://www.pombase.org/assets/graphical_abstract/2021-01-11-33506191-Yukawa-escape-from-catastrophe.png)

This is a downstream consequence of failure of chrosome segregation: If cytokinesis occurs, the nucleus will be inherited by one of the two daughter cells, but this cell will likely be sick (or at least diploid). I propose to either:
* Delete these annotations, as they are already annotated to failed mitosis
* Label as spontaneous diploidization (but this will be also true for many other thermosensitive mutants where mitosis fails).

Let me know what you think is best.

The remaining annotation has 2.1% penetrance and comes from imp1D. Defined in PMID:15937127 as:
> Mono-nucleated cells in which the nucleus is not in the center of the cell, a phenotype never seen in wild-type cells, represented 2.1% of the imp1Δ population.

I think we can safely annotate imp1D it to: mislocalized nucleus during vegetative growth (FYPO:0002071)

If you would agree with these changes, the term would be orphaned, and I can create a ticket to obsolete it.

**Comment by manulera** [comment]
* [ ] Something on double phenotypes

**Comment by ValWood** [comment]
Yes  we decided to remove these annotations as the spontaneous diploidization (as it is a different way of looking at the same phenotype?). However  only some alleles have this annotation.

maybe I should also add a comment to "[increased spontaneous diploidization](https://www.pombase.org/term/FYPO:0000252)"
to say FYPO:0000252 sometimes  occurs when  the nucleus is displaced from the cell centre by actin cables? 

Note that there is also a descendant term
"septated mononucleate vegetative cell with mislocalized nucleus and anucleate compartment FYPO:0007646"

which could also merge into "spontaneous diploidization" it seems?
do you agree?

**Comment by manulera** [comment]
## Regarding single allele phenotype

> maybe I should also add a comment to "increased spontaneous diploidization"
to say FYPO:0000252 sometimes occurs when the nucleus is displaced from the cell centre by actin cables?

Either this, or we could also add a child to `increased spontaneous diploidization`:

 `increased spontaneous diploidization due to abolished sister chromatid separation`. There, we could specify "the nucleus is displaced from the cell center by actin cables, preventing chromosome cut and leading to diploidisation: PMID:33506191". This new term could be a child of:

* FYPO:0001946 - 'abolished mitotic sister chromatid separation'
* FYPO:0000252 - 'increased spontaneous diploidization'

Also here there is again the use of `increased` for a yes/no outcome. Unless this is because there is a baseline of spontaneous diplodisation that is considered, it could probably be renamed to `spontaneous diploidization` with a given penetrance.

> Note that there is also a descendant term
"septated mononucleate vegetative cell with mislocalized nucleus and anucleate compartment FYPO:0007646"

I can revisit these annotations and see whether they also refer to diploidisations.

## Regarding double allele phenotype

![Screenshot 2022-05-16 at 09 50 28](https://user-images.githubusercontent.com/22526102/168544617-116a7485-77d2-48d2-a628-f2954e6fcba0.png)

A comment on the annotated double phenotypes:

The double phenotypes are the combined effects of failing mitosis and failing cytokinesis/septation as consequence of disrupting actin cytoskeleton, but the failure in cytokinesis is independently caused by the allele that is not cut7-22:

* No segregation due to previous allele -> mononucleate cell.
* No cytokinesis due to extra allele -> aseptate cell.

For example:

* cdc12-112: [inviable elongated multinucleate vegetative cell with abnormal septum morphology](https://www.pombase.org/term/FYPO:0002025)
* cdc7-24: abolished cytokinesis

The phenotype of not being able to move the nucleus away from the center of the cell is interesting though, so we could move the double mutants to a new term:

Abolished nuclear displacement from septation site by actin cables after abolished mitotic sister chromatid separation


This new term could be a child of:

* FYPO:0001946 - 'abolished mitotic sister chromatid separation'
* FYPO:0002435 - 'abnormal actin cables'

**Comment by ValWood** [comment]
I think the child term is a good idea. It will be useful to know if there are different mechanisms for this phenotype to occur. Could you open a FYPO tracker ticket for that one.

**Comment by manulera** [comment]
* [ ] After moving the existing annotations, children terms of FYPO:0006339 should be revisited, as annotations can probably be reassigned to existing terms, or terms can be made children of other terms.

**Comment by manulera** [comment]
* I moved the annotations from the double deletion from FYPO:0005321 to FYPO:0007966
* The question remains as to what to do with single allele phenotypes that describe the visible effects of diploidisation:
  * "septated mononucleate vegetative cell with mislocalized nucleus and anucleate compartment FYPO:0007646"
  * "mononucleate vegetative cell with mislocalized nucleus" - at least in some of its uses, where it does not mean FYPO:0002069 (FYPO:0002069)

I think this comes back to one of those cases of cause vs. consequence / process vs. outcome

**Comment by ValWood** [comment]
I think it is probably OK to recorded these phenotypes, because although we know this is a consequence of the diploidization, this is the observed phenotype.  But if this phenotype is *only* caused by spontaneous diploidization there can be a link in the ontology (I forget which relationship this is has_output or output_of...we will need to look in the ontology, or old tickets  to see how they are used, I have not yet seen any in Protege).
These relationships will allow the annotations to 
"mononucleate vegetative cell with mislocalized nucleus"  to be retrieved with a query on "increased spontaneous diploidization"

**Comment by ValWood** [comment]
ignore previous comment. I agree these should be spontaneous diploidisation. You just need to check that all the descendants also represent mitotic phenotypes. I was not sure about
 
"inviable elongated mononucleate vegetative cell with mislocalized nucleus"
(but mislocalized nucleus is probably not critical to capture here?)




[inviable elongated mononucleate vegetative cell with mislocalized nucleus](https://www.pombase.org/term/FYPO:0002337)

**Comment by manulera** [comment]
Yes let's discuss this next time
